Makefile

```diff
@@ -8,6 +8,12 @@ install-cli: compile
 	@echo " > \033[32mInstalling cb-sol-cli... \033[0m "
 	npm link ./cli 
 
+install-celo-ganache:
+	git clone https://github.com/celo-org/ganache-cli.git || cd ./ganache-cli/; git pull
+	npm install --prefix ./ganache-cli
+	ln -f -s  ~/chainbridge-solidity/ganache-cli/cli.js  ~/.local/bin
```

Comment:
Had to make it an absolute path for symbolic link to work, but it also assumes that your chainbridge-solidity directory is located within home directory.

Reply:
Use `$PWD` instead of this path